### PR TITLE
maintenance: bumped to latest utils dependency version

### PR DESCRIPTION
**Pull request by jimleroyer**
## Issue 

This is in the context of this task:
https://trello.com/c/62gscLJH/169-graph-the-services-going-over-the-daily-rate-limit

## Change

As I upgraded the *notification-utils* project, both *admin* and *api* need to get bumped to the latest version.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -49,7 +49,7 @@ awscli==1.18.188
 bleach==3.2.1
 botocore==1.19.28
 cachetools==4.1.1
-certifi==2020.11.8
+certifi==2020.12.5
```
Comment (brdunfield) [inline requirements.txt:52]: Is this change related to making utils work?

**Review by brdunfield — APPROVED** [review]
👍

**Review by RenDavidDimen — APPROVED** [review]
Looks good to me! I'm also curious with what Bethany had asked before

File: requirements.txt (hunk 1 above)
Comment (jimleroyer) [reply requirements.txt:52]: No, this was a 'natural' bump after running `make freeze`.
